src/formio/components/address.ts

```diff
@@ -0,0 +1,22 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export interface AddressData {
+  postcode: string;
+  housenumber: string;
+  houseletter?: string;
+  housenumberaddition?: string;
+}
+
+export type AddressInputSchema = InputComponentSchema<AddressData, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Concrete types
+ */
+export interface AddressComponentSchema
+  extends Omit<AddressInputSchema, 'hideLabel' | 'placeholder' | 'disabled'> {
```

Comment:
`validateOn` is still a valid property, maybe we should drop that key too since the component is a composite anyway and we will likely hardcode this behaviour.